Without this, the FILE reference in this header can cause compile issues.

RP - 29/1/10

Upstream-Status: Accepted

Index: trunk/libopkg/pkg_dest.h
===================================================================
--- trunk.orig/libopkg/pkg_dest.h	2010-01-29 09:37:22.000000000 +0000
+++ trunk/libopkg/pkg_dest.h	2010-01-29 09:37:33.000000000 +0000
@@ -18,6 +18,8 @@
 #ifndef PKG_DEST_H
 #define PKG_DEST_H
 
+#include <stdio.h>
+
 typedef struct pkg_dest pkg_dest_t;
 struct pkg_dest
 {
